README.md

```diff
@@ -1,9 +1,57 @@
-# TIC TAC TOE
+## TIC TAC TOE
 This is the third project of the Main Ruby curriculum at Microverse. The goal is to Build a tic-tac-toe game on the command line where two human players can play against each other and the board is displayed in between turns.
+* [Authors](#authors)
+* [Technologies](#technologies)
+* [Setup](#setup)
+* [Instructions](#instructions)
+* [Winnning Patterns](#winning-patterns)
 
-# Authors
-## Raymond Oluoch https://github.com/rOluochKe/
-## Andres F. Moya https://github.com/AndresFMoya/
+## Authors
+This project was contibuted by
+* Raymond Oluoch https://github.com/rOluochKe/
```

Comment:
Using a format like:

Raymond Oluoch - [Github](https://github.com/rOluochKe/)

```
Raymond Oluoch - [Github](https://github.com/rOluochKe/)
```

On my opinion, would look even better, but that is a personal preference.